api/interfaces/CommentsInterfaces.ts

```diff
@@ -0,0 +1,457 @@
+﻿/*
```

Comment:
Where did we get this file from? Looks new.

Reply:
It is new, it was necessary for the build to succeed though, some WIT files were depending on it